support/docker/production/Dockerfile.arm

```diff
@@ -0,0 +1,65 @@
+FROM arm64v8/debian:stretch-slim
+
+RUN set -ex; \
+    if ! command -v gpg > /dev/null; then \
+      apt-get update; \
+      apt-get install -y --no-install-recommends \
+        gnupg \
+        dirmngr \
+      ; \
+      rm -rf /var/lib/apt/lists/*; \
+fi
+
+# install nodejs and npm
+ENV DEBIAN_FRONTEND noninteractive
+RUN apt-get update \
+&& apt-get install -y apt-utils nano htop curl git build-essential gcc g++ make \
+&& curl -sL https://deb.nodesource.com/setup_8.x | bash - \
+&& apt-get install -yq nodejs \
+&& rm -rf /var/lib/apt/lists/ \
+&& npm install npm@latest -g && npm install yarn@latest -g
```

Comment:
npm ins't usefull for peertube, you can do evrythings with yarn.
It comes with nodejs package but I think its beter to follow https://yarnpkg.com/lang/fr/docs/install/#debian-stable
And only update yarn. Upgrading npm is a waste of time and space.
(that not a full change request, more a comment)

Reply:
Keep that in mind for future image optimization.
This one is not perfect but I will improve it later.

Thanks

Reply:
Yes, that here gaining somethings like 30sec at best and probably 10mb at best, so no need to press :) .

Reply:
Anyway I will test each of this improvement on both ARM and x86 in the coming days.

Reply:
In this case npm is installed with nodejs so we will just make sure we got the latest version